.git_hooks/hooks/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+[ -n "$THOTH_NO_PRECOMMIT_TEST" ] && exit 0
+
+which pipenv > /dev/null 2>&1 || {
+RED='\033[0;31m'
```

Comment:
Why was the indentation removed here and bellow?

Reply:
Had to rewrite the whole file, was deleted due to `git checkout .` :see_no_evil:  :(
I have re-edited the changes have a look, please.